docs/api_documentation.md

```diff
@@ -573,11 +571,6 @@ See [SCIMv2 specification]
 (http://tools.ietf.org/html/draft-ietf-scim-api-02#section-3.2.1) for further
 details.
 
-#### Get the User of the current accessToken
```

Comment:
Please replace instead of delete the documentation in form of [`Get a single User`](https://github.com/osiam/osiam/blob/master/docs/api_documentation.md#get-a-single-user)